docs/source/upgrade-guide/index.md

```diff
@@ -1047,15 +1047,9 @@ You can consider removing it if you were shadowing it in your project.
 ### Grid block migration from `@kitconcept/volto-blocks-grid`
 
 ```{versionadded} Volto 17.0.0-alpha.16
+  It is based on the @kitconcept/volto-blocks-grid add-on version 7.x.x..
```

Comment:
Please revert this change. We often want this empty, especially when the context is clear. Explanation should remain in the narrative text below.

```suggestion
```